guide/14-deep-learning/inference-only-text-models-in-arcgis-learn.ipynb

```diff
@@ -0,0 +1,1001 @@
+{
```

Comment:
<ol><li>Hyperlink to a resource explaining masked language modeling would be helpful to the user.</li><li><strong>FillMask</strong> model can be used to provide suggestion for a <s>given</s> <strong>missing(?) </strong>token/word in a sentence.</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/851/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>

Reply:
<p>fixed</p>

Reply:
<p>https://huggingface.co/transformers/task_summary.html#masked-language-modeling </p><p>Can we use the above link to explain masked language modeling?</p>

Reply:
done